app/Notifications/Match/JoinMatchRequest.php

```diff
@@ -77,4 +79,11 @@ public function toArray($notifiable): array {
 			//
 		];
 	}
+
+	public function toConversation($notifiable)
+    {
```

Comment:
adda referance to the match here - so we can show a link to which match he talks about